src/main/java/com/google/sps/data/IAMRecommenderMetadata.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.data;
+
+import com.google.sps.data.Recommendation.RecommenderType;
+
+/** Class that provides additional information about IAM Binding recommendations. */
+public class IAMRecommenderMetadata extends RecommenderMetadata {
+
+  public int impactInIAMBindings;
```

Comment:
Could you please provide a link to this impact? E.g. Is it from Recommender API, etc?

Reply:
done!